src/Tools/ConfigurationSchemaGenerator/RuntimeSource/Configuration.Binder/Parser/ConfigurationBinder.cs

```diff
@@ -1,9 +1,7 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
```

Comment:
It might be useful to skip all the unrelated changes in this PR, so it is more reviewable.